solutions/gold/usaco-863.mdx

```diff
@@ -7,6 +7,21 @@ author: Sofia Yang, Ryan Chou
 
 [Official Analysis (C++)](http://www.usaco.org/current/data/sol_teamwork_gold_dec18.html)
 
+
+<Spoiler title="Hint 1">
```

Comment:
im ngl these are barely hints

Reply:
in terms of "no crap" or "gives you the solution"

Reply:
no crap

Reply:
99.98% of users at this level would already think of NK and not some other weird obscure complexity.
```Meaning, for each of the $N$ cows, we need to calculate something for them $K$ times.```
Damn thats crazy, at least specify "something"

Reply:
How about now?

Reply:
how about
`we can recalculate the most optimal sums of skill levels $K$ times`?

Reply:
> how about `we can recalculate the most optimal sums of skill levels $K$ times`?

oh ok, how about now